docs/one-click-integrations/github-integration.md

```diff
@@ -115,12 +140,26 @@ The table below lists the data that the GitHub integration collects from your Gi
     <td>
         <p>Deployments:</p>
         <ul>
-            <li><code>deploy_id</code>: pull request ID</li>
+            <li><code>deploy_id</code>: pull request identifier</li>
+            <li><code>timestamp_created</code>: pull request merged date</li>
             <li><code>system</code>: repository name</li>
         </ul>
     </td>
     <td>Deployment frequency and Change failure rate on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
 </tr>
+<tr>
+<td>Pull requests</td>
+    <td>
+        <p>Incidents:</p>
+        <ul>
+            <li><code>incident_id</code>: pull request number</li>
+            <li><code>timestamp_created</code>: reverted pull request merged date</li>
```

Comment:
I'd. break up this [noun string](https://www.plainlanguage.gov/guidelines/words/avoid-noun-strings/)

```suggestion
            <li><code>timestamp_created</code>: merge date of the reverted pull request</li>
```